package.json

```diff
@@ -36,7 +36,7 @@
     "glob": "^7.1.6",
     "react-calendar": "^3.3.1",
     "react-paginate": "^7.0.0",
-    "react-tooltip-lite": "^1.12.0",
+    "react-tooltip-lite": "npm:@carforyou/react-tooltip-lite@1.12.1",
```

Comment:
```suggestion
    "@carforyou/react-tooltip-lite": "1.12.1",
```

Reply:
this does not work because the `declare module` is named differently :D

Reply:
uuuh. then I fucked up with the release. this should be very temporary, so let's not sweat it